libs/core/src/lib/tabs/tab-utils/tab-directives.ts

```diff
@@ -126,70 +125,61 @@ export class TabIconDirective implements CssClassBuilder, OnInit {
     // tslint:disable-next-line:directive-selector
     selector: '[fd-tab-tag]',
     host: {
-        'class': 'fd-tabs__tag'
-    }
+        class: 'fd-tabs__tag',
+    },
```

Comment:
IT's really detail, but I see that in `tab-directives` there are a lot of hanging commas. Could you remove them ?

Reply:
removed